raiden/network/transport/matrix/transport.py

```diff
@@ -842,7 +851,10 @@ def _handle_message(self, room, event) -> bool:
         for message in messages:
             if isinstance(message, Delivered):
                 self._receive_delivered(message)
+            elif isinstance(message, Processed):
```

Comment:
Is this distinction necessary? `Processed` is itself a `SignedRetrieableMessage`